yarn.lock

```diff
@@ -10713,6 +10713,11 @@ vue-loader@^15.7.0:
     vue-hot-reload-api "^2.3.0"
     vue-style-loader "^4.1.0"
 
```

Comment:
Ok, I see the folly of doing the review commit-by-commit. I'm going to back out and look at all the code at once to spare you my unnecessary comments moving forward.

Reply:
Ah yep. I generally work in the sense that I might make a bunch of commits as I'm working on things with the intention of squashing and merging it all in at the end.

I prefer to keep the lock file in git so that when someone pulls down the code, they can get the exact versions that was being used last.

A bit long, but some info here
https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

Reply:
Oh, _interesting_.... I'm going to have to save putting this fully into my brain for later. Thank you.